Source/Extensions/Blazorise.Charts.Annotation/wwwroot/chart.annotation.js

```diff
@@ -0,0 +1,36 @@
+import { getChart } from "../Blazorise.Charts/charts.js?v=1.2.2.0";
+import { parseFunction } from "../Blazorise.Charts/utilities.js?v=1.2.2.0";
```

Comment:
we already have an open issue for v1.3 for this, but we need to update to v1.3. Just friendly reminder.